### PR TITLE
Add xrft recipe

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by kynan** [comment]
Good to got @conda-forge/core ?

File: recipes/xrft/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = "0.1.dev" %}
+{% set sha256 = "b7235736b0188feeb8d25d3368ac5f7eb0d7d38129080687e82ee906b587b69a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rabernat/{{ name }}/archive/master.tar.gz
```
Comment (jakirkham) [inline recipes/xrft/meta.yaml]: So this isn't really a tag and it is just building from `master`. 😕 Can you ask upstream to add a tag and use that instead?

Comment (kynan) [inline recipes/xrft/meta.yaml]: This is just temporary :) @rabernat are you happy to tag a release?

Comment (kynan) [inline recipes/xrft/meta.yaml]: But thanks for pointing this out @jakirkham - reassuring to know you thoroughly vet all those recipes :slightly_smiling_face:

Comment (rabernat) [inline recipes/xrft/meta.yaml]: Sorry for the delay...I just tagged release v0.1 of xrft.

**Pull request merged**